dotCMS/src/main/java/com/dotcms/content/business/json/ContentletJsonAPIImpl.java

```diff
@@ -313,7 +314,9 @@ private Map<String, Object> getContentletMapFromImmutable(final Contentlet immut
             if(value instanceof Date){
                 value = new Timestamp(((Date) value).getTime());
             }
-
+            if (field instanceof StoryBlockField) {
```

Comment:
may add some comment here why we are adding the raw, in a year nobody will remember why we are doing it